api/routes/api.js

```diff
@@ -13,27 +13,27 @@ const GetUsersDevicesSchema = {
 	required: ['user_id'],
 	properties: {
 		user_id: {
-			type: 'string'
+			type: 'string' 
 		}
 	}
 };
 
 const GetDeviceDataSchema = {
 	type: 'object',
-	required: ['device_id'],
+	required: ['device_id', 'data_run'],
```

Comment:
Is the UI app updated to send the data_run it wants? Or are we not checking this schema anywhere?